docs/changelog/83665.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 83665
+summary: "RemoveProcessor updated to support fieldsToKeep"
+area: Infra/Scripting
```

Comment:
Technically, this falls into the "ingest" category:
```suggestion
area: Ingest
```

Reply:
ok